data/crac/crac-api/src/main/java/com/farao_community/farao/data/crac_api/RemedialAction.java

```diff
@@ -55,18 +55,10 @@ public interface RemedialAction<I extends RemedialAction<I>> extends Identifiabl
      */
     Optional<Integer> getSpeed();
 
-    /**
-     * Evaluates if the remedial action is available depending on its UsageMethod.
-     */
-    boolean isRemedialActionAvailable(State state);
-
-    /**
-     * Evaluates if the remedial action is available depending on its UsageMethod.
-     */
-    boolean isRemedialActionAvailable(State state, boolean evaluatedCondition);
-
     Set<FlowCnec> getFlowCnecsConstrainingUsageRules(Set<FlowCnec> perimeterCnecs, Network network, State optimizedState);
 
+    Set<FlowCnec> getFlowCnecsConstrainingForOneUsageRule(UsageRule usageRule, Set<FlowCnec> perimeterCnecs, Network network);
```

Comment:
I think these methods belong in RaoUtil (you will need to add the RA as an argument of the first). And they are pretty ugly IMO, they may well deserve a refactoring, but we can do that later